judge/migrations/0125_add_voting_functionality.py

```diff
@@ -0,0 +1,40 @@
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('judge', '0124_contest_show_short_display'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='profile',
+            name='is_banned_problem_voting',
+            field=models.BooleanField(default=False,
+                                      help_text="User will not be able to vote on problems' point values.",
+                                      verbose_name='banned from voting'),
+        ),
+        migrations.CreateModel(
+            name='ProblemPointsVote',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('points', models.IntegerField(help_text='The amount of points you think this problem deserves.',
+                                               validators=[django.core.validators.MinValueValidator(1),
+                                                           django.core.validators.MaxValueValidator(50)],
```

Comment:
I think an upper bound is reasonable. Otherwise, trolls can put `100000000000000000000000...` and completely mess up the graphs. Other DMOJ clones can change the setting to change the upper bound.

Reply:
I think we capped this in the js code.